docs/basics/paintbrush.md

```diff
@@ -37,3 +37,8 @@ Clicking the **Use 3D brushstrokes** button will activate the 3D brush, 3D annot
 The transparency of active and non-active annotations can be altered by clicking the **Annotation Transparency** button.
 Using the two sliders allows you to modify the transparency of both the current (brush) and all other (brush) annotations in the range of 0-100% (opacity).
 This can be a very useful tool to determine the accuracy of drawn annotations by comparing to the background image.
+
+## Strokes vs. Pixels
+
+Brush strokes can be converted from the standard vector-graphic representation into pixel level annotations, based on the size of the pixels of the image being annotated.
+To convert, click the **Show strokes as pixels** button.
```

Comment:
Either we need a specific image of this button, or some text labels on the summary image at the top of the page